packages/types/src/span.ts

```diff
@@ -88,7 +88,7 @@ export interface SpanContext {
   /**
    * Completion status of the Span.
    */
-  status?: SpanStatus;
```

Comment:
This is, of course, a downside but given we save `0.3kb` gzipped I think it's worth it.